tests/tt_eager/python_api_testing/unit_testing/misc/test_softmax.py

```diff
@@ -21,42 +23,44 @@
 
 @pytest.mark.parametrize(
     "dtype",
-    (ttl.tensor.DataType.BFLOAT16, ttl.tensor.DataType.FLOAT32),
+    (ttnn.bfloat16, ttnn.float32),
     ids=["bfloat16", "float"],
 )
 @pytest.mark.parametrize("inplace", [True, False])
 def test_softmax(device, inplace, dtype):
-    if is_grayskull() and dtype == ttl.tensor.DataType.FLOAT32:
+    if is_grayskull() and dtype == ttnn.float32:
         pytest.skip("Skipping float32 tests on Grayskull")
 
     torch.manual_seed(0)
-    sm_op = ttl.operations.primary.softmax_in_place if inplace else ttl.tensor.softmax
+    sm_op = ttnn.softmax_in_place if inplace else ttnn.softmax
 
     input_shapes = [(3, 64, 128, 96), (1, 64, 32, 32)]
 
     for input_shape in input_shapes:
         input_tensor = torch.randn(input_shape).bfloat16()
 
-        tt_input_tensor = ttl.tensor.Tensor(input_tensor, dtype).to(ttl.tensor.Layout.TILE).to(device)
+        tt_input_tensor = ttnn.from_torch(input_tensor, dtype=dtype, layout=ttnn.TILE_LAYOUT, device=device)
 
         if not is_grayskull():
-            if dtype == ttl.tensor.DataType.FLOAT32:
-                compute_kernel_config = ttl.tensor.WormholeComputeKernelConfig(
-                    math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+            if dtype == ttnn.float32:
+                compute_kernel_config = ttnn.WormholeComputeKernelConfig(
+                    math_fidelity=ttnn.MathFidelity.HiFi4,
                     math_approx_mode=False,
                     fp32_dest_acc_en=True,
                 )
             else:
-                compute_kernel_config = ttl.tensor.WormholeComputeKernelConfig(
-                    math_fidelity=ttl.tensor.MathFidelity.HiFi4,
+                compute_kernel_config = ttnn.WormholeComputeKernelConfig(
+                    math_fidelity=ttnn.MathFidelity.HiFi4,
                     math_approx_mode=False,
                     fp32_dest_acc_en=False,
                 )
 
         tt_output_tensor_on_device = sm_op(
             tt_input_tensor, compute_kernel_config=compute_kernel_config if not is_grayskull() else None
         )
-        tt_output_tensor = tt_output_tensor_on_device.cpu().to(ttl.tensor.Layout.ROW_MAJOR).to_torch()
+        tt_output_tensor = ttnn.to_layout(tt_output_tensor_on_device, ttnn.ROW_MAJOR_LAYOUT)
+        tt_output_tensor = ttnn.from_device(tt_output_tensor)
+        tt_output_tensor = ttnn.to_torch(tt_output_tensor)
```

Comment:
I think this can be simplified to `to_torch`

Reply:
will try